src/style.css

```diff
@@ -2,0 +1,98 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900px;
+}
+
+* {
+  font-family: Avenir-Book, sans-serif;
+  font-weight: 300;
+  font-style: normal;
+  color: #3d4e61;
+
+  box-sizing: border-box;
+}
+
+/* .page {
+  font-family: Avenir-Book, sans-serif;
+  font-weight: 300;
+  color: #3D4E61;
+
+  font-style: normal;
+} */
+
+.page__searchBar {
+  margin-top: 20px;
+  position: relative;
+}
+
+.searchBar {
+  background: #fff;
+}
+
+.searchBar__input {
+  /* box-sizing: border-box; */
```

Comment:
The comment `/* box-sizing: border-box; */` is redundant since `box-sizing: border-box;` is already applied globally in the universal selector `*`. You can remove this comment.